modules/tinymce/src/core/main/ts/init/ContentCss.ts

```diff
@@ -2,9 +2,14 @@ import { Arr } from '@ephox/katamari';
 
 import Editor from '../api/Editor';
 import * as Options from '../api/Options';
+import { TinyMCE } from '../api/Tinymce';
+
+declare let tinymce: TinyMCE;
 
 const isContentCssSkinName = (url: string) => /^[a-z0-9\-]+$/i.test(url);
 
+const isBundledCssSkinName = (url: string) => tinymce.Resource.has('content/' + url + '/content.css');
```

Comment:
This `'content/' + name + '/content.css'` concept probably needs a reusable home, it's hard coded 3 times in this PR alone

Reply:
Done!